.github/workflows/pipeline.yml

```diff
@@ -39,6 +39,13 @@ jobs:
           args: --all-features --workspace
       - name: Check snapshots
         run: git diff --exit-code -- tests/out
+      - uses: actions-rs/cargo@v1
+        name: Test all (without validation)
```

Comment:
why do we test all without validation?

Reply:
We need this to make sure, that backend generates exactly the same shader without the validation feature. `Naga` is collecting `ModuleInfo` inside the validation module, so, theoretically, we can accidentally break shader generation.

Reply:
Can this be done in Lazy CI? Chance that it breaks are rather low.